packages/dom-helper/lib/main.js

```diff
@@ -567,13 +569,26 @@ prototype.parseHTML = function(html, contextualElement) {
 var parsingNode;
 
 // Used to determine whether a URL needs to be sanitized.
-prototype.protocolForURL = function(url) {
-  if (!parsingNode) {
-    parsingNode = this.document.createElement('a');
-  }
+// In Node.js environments, we can't rely on the DOM correctly parsing URLs.
+// Instead, we'll fall back to Node's built-in URL parsing.
+if (typeof process === 'object' && process + '' === '[object process]') {
```

Comment:
Unknown, but looks like this might be a better test: https://github.com/jnath/nwjs-video-player/blob/159c0d487159846664782979d46bbd61857fb429/src/app/js/nw.js#L1